src/components/ProjectSortSelector.tsx

```diff
@@ -1,6 +1,28 @@
+import { t } from 'i18next';
 import { useTranslation } from 'react-i18next';
+import { ProjectSearchOrder } from '../api/cristinApi';
+import { SortOrder } from '../api/searchApi';
 import { SortSelector } from './SortSelector';
 
+export interface ProjectSortOption {
+  orderBy: ProjectSearchOrder;
+  sortOrder: SortOrder;
+  label: string;
+}
+
+export const projectSortOptions: ProjectSortOption[] = [
+  {
+    orderBy: ProjectSearchOrder.Name,
+    sortOrder: 'asc',
+    label: t('search.sort_alphabetically_asc'),
```

Comment:
Burde ha i18n key istedet for translation, sånn at det alltid er ajour med current språk?